docs/user/how-to/run-cli.rst

```diff
@@ -0,0 +1,80 @@
+Control the Worker via the CLI
+==============================
+
+The worker comes with a minimal CLI client for basic control. It should be noted that this is 
+a test/development/debugging tool and not meant for production!
+
+.. seealso:: 
+    You must have access to the ``blueapi`` command either via the container or pip installing.
+    `./run-container` and `../tutorials/installation`
+
+
+.. seealso:: 
+    In a developer environment, the worker can also be run from vscode: `../../developer/tutorials/dev-run`.
+
+
+Start ActiveMQ
+--------------
+
+The worker requires a running instance of ActiveMQ, the simplest
+way to start it is to run it via a container:
+
+.. tab-set::
+
+    .. tab-item:: Docker
+
+        .. code:: shell
+
+            docker run -it --rm --net host rmohr/activemq:5.15.9-alpine
+
+    .. tab-item:: Podman
+
+        .. code:: shell
+
+            podman run -it --rm --net host rmohr/activemq:5.15.9-alpine
+
+
+Start the Worker
+----------------
+
+To start the worker
+
+.. code:: shell
+
+    blueapi serve
```

Comment:
might be worth mentioning about the config. I've just realised that, unfortunately the following does not work:

```
blueapi --config some/file.yaml
blueapi serve
```
i.e. the `blueapi serve` command will not use the configuration from the first command. Is this a problem at all? You would need to do;

```
blueapi --config some/path.yaml serve
```

For config to be ingested to the server.

Reply:
Done in quickstart guide

Reply:
That's the behaviour I'd expect. I think it would be odd if the config persisted from one call to the next.

Reply:
Agreed